client/src/components/shared/calcStarImg.jsx

```diff
@@ -5,6 +5,7 @@ const calcStarImg = (numStars) => {
   let wholeStars = Math.floor(numStars);
   let fractionStar = numStars - Math.floor(numStars);
   let emptyStars = 5 - Math.ceil(numStars);
+  if (fractionStar > 0 && fractionStar < 0.25) { emptyStars++; }
```

Comment:
@the-trash-panda It turns out that the star image calculation works if the product has at least a fractionStar that's at least a quarter. For example: 3.6 rating = 3 whole stars + 1 fraction star + 1 empty star.

But if the product has a fractionStar that's less than a quarter, then it won't have enough stars. For example: 4.1 rating = 4 whole stars + 0 fraction stars + 0 empty stars.

So I added this line here to add another empty star

Reply:
Nice catch, thanks for the explanation!